### PR TITLE
Problem: ruby client is not tested

### PR DESCRIPTION
**Pull request by dkliban**
Solution: run the .travis/test_bindings.rb with each PR

re: #4896
https://pulp.plan.io/issues/4896

### CODE REVIEW

File: .travis/script.sh [hunk 1]
```diff
@@ -65,6 +65,23 @@ if [ "$TEST" = 'bindings' ]; then
   ./generate.sh pulp_file python
   pip install ./pulp_file-client
   python $TRAVIS_BUILD_DIR/.travis/test_bindings.py
+  if [ ! -f $TRAVIS_BUILD_DIR/.travis/test_bindings.rb ]
```
Comment (daviddavis) [inline .travis/script.sh]: What's the purpose of this check?

File: .travis/script.sh [hunk 2]
```diff
@@ -65,6 +65,23 @@ if [ "$TEST" = 'bindings' ]; then
...
+  gem build pulpcore_client
+  gem install --both ./pulpcore_client-0.gem
+  cd ..
+  rm -rf ./pulp_file-client
+  ./generate.sh pulp_file ruby
+  cd pulp_file-client
+  gem build pulp_file_client
+  gem install --both ./pulp_file_client-0.gem
+  cd ..
+  ruby $TRAVIS_BUILD_DIR/.travis/test_bindings.rb
```
Comment (daviddavis) [inline .travis/script.sh]: This is kind of hard to read. Can you organize this by adding in some line breaks?

File: .travis/script.sh (hunk 1 above)
Comment (dkliban) [reply .travis/script.sh]: It only tests ruby bindings if the file exists. I will open a PR to plugin_template with this change.

File: .travis/script.sh (hunk 2 above)
Comment (dkliban) [reply .travis/script.sh]: Sure. I'll do it.

**Pull request merged**